private_gpt/components/ingest/ingest_helper.py

```diff
@@ -48,7 +52,10 @@ def _try_loading_included_file_formats() -> dict[str, type[BaseReader]]:
         ".mbox": MboxReader,
         ".ipynb": IPYNBReader,
     }
-    return default_file_reader_cls
+    optional_file_reader_cls: dict[str, type[BaseReader]] = {
```

Comment:
I think you can move it back with the default readers, you are importing it unconditionally anyway